### ISSUE TITLE
remove old jobs

### ISSUE THREAD

**Issue by schneefux** [vainglorygame/joblib#14]
Delete jobs that are finished

**Comment by kvahuja** [comment]
yes makes sense. run a cron to remove all finished (not failed) jobs from DB. the job can run every hour day and clean up
